README.md

```diff
@@ -32,7 +32,51 @@ Features of Clash:
 
 **IRC**: [freenode#clash-lang](https://webchat.freenode.net/#clash-lang)
 
-# Using Clash from source
+# Installing Clash
+
+## Installing via Snap
+
+Clash is released as a binary package on snapcraft. Snap is supported on all
+major Linux distributions. Visit [Clash’s snapcraft page](https://snapcraft.io/clash),
+scroll down, and choose your distribution for installation instructions. To
+install the latest stable version, use:
+
+```bash
+snap install clash
+```
+
+To install the latest development version of Clash, run:
+
+```bash
+snap install clash --edge
+```
+
+This version is updated every 24 hours.
+
+## Installing via Source (Linux / MacOS)
+
+Install the [latest nix](https://nixos.org/nix/download.html) and run:
+
+```bash
+curl -s -L https://github.com/clash-lang/clash-compiler/archive/1.0.1.tar.gz | tar xz
```

Comment:
`1.0.1.tar.gz`: if you keep this at `1.0` it will use the branch 1.0 which includes all the latest fixes. I'm afraid we're going to forget to update this `README.md` on new releases..

Reply:
Your call.

Reply:
Ah good point, I assumed 1.0 would always point to 1.0 instead of 1.x. I'll change it back